sdk/python/packages/flet/src/flet/cli/commands/build.py

```diff
@@ -251,6 +251,22 @@ def add_arguments(self, parser: argparse.ArgumentParser) -> None:
             help="the branch, tag or commit ID to checkout after cloning the repository with Flutter bootstrap template",
         )
 
+    # according to build matrix given at: 
+    # https://flet.dev/docs/guides/python/packaging-app-for-distribution/#build-platform-matrix
+    @staticmethod
+    def _can_build_target_platform(target_platform: str) -> bool:
+        current_platform = platform.system()
+
+        allowed_builds = {
+            'Darwin': ['ipa', 'apk', 'aab', 'macos', 'web'],  # macos
+            'Windows': ['apk', 'aab', 'linux', 'windows', 'web'],
+            'Linux': ['apk', 'aab', 'linux', 'web']
+        }
```

Comment:
Hey, thanks for the update!

I've been thinking though about adding "build_platrofms" to the existing `self.platforms`, to keep everything in a single place.

Kind of:

```python
...
            "apk": {
                "build_command": "apk",
                "status_text": ".apk for Android",
                "output": "build/app/outputs/flutter-apk/*",
                "dist": "apk",
                "can_be_run_on": ["Darwin", "Windows", "Linux"]
            },
...

what do you think?
```

Reply:
edit `"can_run_on"`.

Reply:
sorry, `can_be_run_on` is more grammatically correct :)

Reply:
yes

Reply:
I agree

Reply:
I have made the necessary changes @FeodorFitsner